src/blocks/card.scss

```
@@ -0,0 +1,80 @@
+.card {
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid var(--card-border-color);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  
+  &__image {
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+  &__title {
```

Add empty line between blocks. Fix everywhere
```suggestion
  }
  
  &__title {
```